os/include/tinyara/net/if/wifi.h

```diff
@@ -6,6 +6,29 @@
 #define TRWIFI_SSID_LEN				  32
 #define TRWIFI_PASSPHRASE_LEN		  64
 
+#define LWNL_POST_WIFIMSG(evt, buffer, buf_len) \
+	lwnl_postmsg(LWNL_DEV_WIFI, evt, buffer, buf_len)
+
+#define LWNL_POST_BLEMSG(evt, buffer, buf_len) \
+	lwnl_postmsg(LWNL_DEV_BLE, evt, buffer, buf_len)
+
+/*  if serialization is failed then driver should let pass the fail event to applications*/
+#define LWNL_POST_WIFISCANMSG(evt, scanlist)							\
+	do {																\
+		if (evt == LWNL_SCAN_DONE) {										\
+			uint8_t *buffer = NULL;										\
+			int lwnl_res = trwifi_serialize_scaninfo(&buffer, scanlist); \
+			if (lwnl_res < 0) {											\
+				LWNL_POST_WIFIMSG(LWNL_SCAN_FAILED, NULL, 0);				\
```

Comment:
In this case, is it okay not to free the `buffer`?

Reply:
yes it's ok